packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2004,6 +2027,9 @@ export const components = ({
             display: "inline-block",
             height: "1em",
             lineHeight: 1,
+            "& svg": {
```

Comment:
setting the font-size to `1em` only for the link with external icon (targeting svg inside MuiLink instead of setting the fontsize for all svgs